jupyter-extension/src/plugins/mount/components/Pipeline/Pipeline.tsx

```diff
@@ -95,7 +95,7 @@ const Pipeline: React.FC<PipelineProps> = ({
           className="pachyderm-pipeline-input-label"
           htmlFor="pipelineName"
         >
-          *Name:{'  '}
```

Comment:
@bbonenfant What does `{' '}` refer to here? is that just for spacing? Also, is the `*` in the description to denote required?

Reply:
Yes to both!